docs/design/error_handling_alignment.md

```
@@ -0,0 +1,358 @@
+
+---
+
+## 5. 互換性の維持
+
+### 5.1 移行戦略
+
+既存の `error_msg()` 関数は、移行期間中も残しておく:
+
+```c
+// 互換性のための簡易ラッパー（deprecated）
+int	error_msg(const char *msg)
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 4

(MD010, no-hard-tabs)